src/components/Lesson/Lesson.scss

```diff
@@ -19,6 +19,6 @@
   @include responsiveFontSizes;
 }
 
-.resetButton {
+.buttonMargin {
```

Comment:
Tror ikke denne brukes lenger.

Reply:
Brukes i LessonButton, ReadmeButton og ResetButton. Tenkte at det er eneste stylingen disse har og alle brukes i Lesson, så jeg lagde ikke tre like scss for hver komponent

Reply:
Men vil det være slik at dersom du endrer stylingen for én av dem, så skal det alltid gjelde for alle tre? I såfall er det kanskje greit (selv om det virker litt random at css-koden ligger i Lesson.scss, burde kanskje vært i en mer generell scss-fil?), men hvis ikke, så burde de som er "uavhengige" ha sin egen fil (selv om koden er lik for alle).

Evt kan du som sagt flytte css-koden i en slags "util"-scss-fil (det går an å inkluderer mer enn 1 scss fil). Men igjen, da vil en endring der endre på alle tre.

Reply:
Laget en scss fil til hver komponent